### ISSUE TITLE
Add utility for Document.from(String query)

### ISSUE THREAD

**Issue by xstefank** [smallrye/smallrye-graphql#869]
Some users might prefer to write queries themselves (like JPQL) but I am just starting with GraphQL so I might have wrong expectations. Currently, I was able to do this:

```java
Document query = document(
            operation(
                field("search(query: \"...\", type: ISSUE, last: 100) {\n" +
                    "    edges {\n" +
                    "      node {\n" +
                    "        ... on PullRequest {\n" +
                    "          number\n" +
                    "          title\n" +
                    "          repository {\n" +
                    "            nameWithOwner\n" +
                    "          }\n" +
                    "          createdAt\n" +
                    "          mergedAt\n" +
                    "          url\n" +
                    "          changedFiles\n" +
                    "          additions\n" +
                    "          deletions\n" +
                    "          labels(last: 100) {\n" +
                    "            edges {\n" +
                    "              node {\n" +
                    "                name\n" +
                    "                color\n" +
                    "              }\n" +
                    "            }\n" +
                    "          }\n" +
                    "        }\n" +
                    "      }\n" +
                    "    }\n" +
                    "  }")
            )
        );
```

So maybe a utility method like this:
```java
Document query = Document.from("search(query: \"...\", type: ISSUE, last: 100) {\n" +
                    "    edges {\n" +
                    "      node {\n" +
                    "        ... on PullRequest {\n" +
                    "          number\n" +
                    "          title\n" +
                    "          repository {\n" +
                    "            nameWithOwner\n" +
                    "          }\n" +
                    "          createdAt\n" +
                    "          mergedAt\n" +
                    "          url\n" +
                    "          changedFiles\n" +
                    "          additions\n" +
                    "          deletions\n" +
                    "          labels(last: 100) {\n" +
                    "            edges {\n" +
                    "              node {\n" +
                    "                name\n" +
                    "                color\n" +
                    "              }\n" +
                    "            }\n" +
                    "          }\n" +
                    "        }\n" +
                    "      }\n" +
                    "    }\n" +
                    "  }");
```

would be a little better to read. Just an idea, feel free to dismiss.

**Comment by jmartisk** [comment]
Why do you need this when you can do `dynamicClient.execute(String query)`?
There is the 
`private Request buildRequest(String query, Map<String, Object> variables, String operationName)`
method in `VertxDynamicGraphQLClient` that turns a string query into a request, but I made it private, as I don't expect anybody using it

**Comment by xstefank** [comment]
@jmartisk I don't see that in quarkus-smallrye-graphql-client 2.0.0.CR3. Closing then.

**Comment by jmartisk** [comment]
Oh right, sorry I didn't mention it. You need the just-released 2.0.0.Final.
